lib/temporal/workflow.rb

```diff
@@ -20,7 +20,7 @@ def self.execute_in_context(context, input)
       Temporal.logger.error("Workflow execution failed", context.metadata.to_h.merge(error: error.inspect))
       Temporal.logger.debug(error.backtrace.join("\n"))
 
-      Temporal::ErrorHandler.handle(error, metadata: context.metadata)
+      Temporal::ErrorHandler.handle(error, context.config, metadata: context.metadata)
```

Comment:
This line is what's breaking the tests, need to dig in a little more to find why.